java-imports.el

```diff
@@ -275,6 +275,35 @@ package and cache it for future statements."
                    (read-string "Class name: "))))
     (java-imports-add-import (s-chop-prefix "@" class))))
 
+;;;###autoload
+(defun java-imports-scan-local-jars (&optional local-repo)
+  (interactive)
+  "scan the local repository, find local jars and add them to the cache.
+if a class is found in multiple packages, any such package may be registered
+for that class in the cache.
+This is currently a synchronous and potentially slow operation, but
+hopefully faster than adding imports manually or using eclipse"
+  (labels ((shell-command-to-lines (cmd)
+				   (s-split "\n"
+					    (shell-command-to-string cmd)
+					    t)))
+    (let* ((local-repo (or local-repo (expand-file-name
+				       "~/.m2/repository/")))
+	   (jars (shell-command-to-lines
+		  (format "find '%s' -name '*.jar'"
+			  local-repo)))
+	   (cache (pcache-repository java-imports-cache-name)))
+      (dolist (jar jars)
+	(dolist (line (shell-command-to-lines (concat "jar -tf " jar)))
+	  (when (string-match "\\(.*\\)/\\(.*\\)[.]class" line)
+	    (let ((class (intern (match-string 2 line)))
+		  (package-name
+		   (->>
```

Comment:
This adds a dependency on the `dash.el` library, can you add that as a dependency to the file?